CRM/Utils/Money.php

```diff
@@ -62,9 +62,12 @@ public static function format($amount, $currency = NULL, $format = NULL, $onlyNu
     }
 
     if ($onlyNumber) {
+      if ($valueFormat === '!%i') {
```

Comment:
this is the default value maybe we should add some deprecation notice for other formats?

Reply:
if that is the case then yes probably